sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/SessionCatalog.scala

```diff
@@ -104,6 +104,13 @@ class SessionCatalog(externalCatalog: ExternalCatalog, conf: CatalystConf) {
     currentDb = db
   }
 
+  // todo: what is the default path in SessionCatalog?
+  def getDefaultPath: String = ""
```

Comment:
this is only used for the database right? I'd remove this and introduce only `getDefaultDBPath`. For `SessionCatalog` I would just use the temp dir.